templates/zerver/confirmation_link_expired_error.html

```diff
@@ -0,0 +1,14 @@
+{% extends "zerver/portico_signup.html" %}
+
+{% block portico_content %}
+<div class="app portico-page">
+    <div class="app-main portico-page-container center-block flex full-page account-creation new-style">
+        <div class="inline-block">
+
+            <div class="get-started">
+                <h1>{{ _("The registration link has expired or is not valid.") }}</h1>
+            </div>
+        </div>
+    </div>
+</div>
+{% endblock %}
```

Comment:
I think HTML like in `templates/zerver/log_into_subdomain_token_invalid.html` shows a better looking error page, maybe we should use that? Can you compare and see what you think? (I'm awful at such decisions :sweat: )

Reply:
I agree. Mainly because of the Login link below the error message.

Reply:
![Captura de tela de 2020-04-28 19-43-53](https://user-images.githubusercontent.com/15564640/80545101-ee03b780-8988-11ea-9136-cccb980c0b7f.png)


![Captura de tela de 2020-04-28 19-44-12](https://user-images.githubusercontent.com/15564640/80545116-f2c86b80-8988-11ea-9a39-63f6c1f17735.png)